app/screens/channel/header/header.tsx

```diff
@@ -46,13 +49,18 @@ type ChannelProps = {
     displayName: string;
     isOwnDirectMessage: boolean;
     memberCount?: number;
-    searchTerm: string;
     teamId: string;
     callsEnabledInChannel: boolean;
     groupCallsAllowed: boolean;
     isTabletView?: boolean;
     shouldRenderBookmarks: boolean;
     shouldRenderChannelBanner: boolean;
+    hasPlaybookRuns: boolean;
+    playbooksActiveRuns: number;
+    isPlaybooksEnabled: boolean;
+    activeRunId?: string;
+
+    // searchTerm: string;
```

Comment:
can be removed

Reply:
This, instead of removed, should be implemented. We are leaving it here for now so we implement it at some point. But yes, it is annoying to have it here.